lib/prometheus_exporter/instrumentation/sidekiq.rb

```diff
@@ -100,7 +100,7 @@ def get_delayed_name(msg, class_name)
           class_name
         end
       end
-    rescue Exception => e
+    rescue
```

Comment:
Is this intended? `rescue` without a specified class will only catch `StandardError` so this is actually an behaviour change

Reply:
Not sure what happened there, the original value was rescue without an exception type which is what it seems to be now in the PR after I reverted the change. I'm guessing I accidentally did a force push, sorry :$